### ISSUE TITLE
TypeError: setting getter-only property "defaultValue"

### ISSUE THREAD

**Issue by unlikenesses** [alpinejs/alpine#221]
I have this component:

```javascript
class Cat {
    constructor(name, age, osc) {
        this.name = name;
        this.age = age;
        this.osc = osc;
    }
}

function foobar() {
    return {
        cats: [],
        init() {
            let osc = audioCtx.createOscillator();
            let cat = new Cat("jim", 42, osc);
            this.cats.push(cat);
        }
    }
}
```

This produces the error:

> TypeError: setting getter-only property "defaultValue"

The problem is the audio oscillator (if I remove that the cat gets pushed fine). Anyone come across this?

**Comment by unlikenesses** [comment]
Hi @SimoTod. Thanks for the feedback. Using the PR certainly half-fixes the issue, but a small issue remains (though I can't for sure say whether it's to do with the proxy). If I push a class with an oscillator instance to an array, then attempt to call `disconnect` on that oscillator, it produces the error

> TypeError: 'disconnect' called on an object that does not implement interface AudioNode.

Codepen here: https://codepen.io/unlikenesses/pen/GRJMzOr

**Comment by calebporzio** [comment]
Thanks for this, but it should be fixed with the new reactivity core in v2.1.0
